crs_application/config/crs.yaml

```diff
@@ -11,9 +11,12 @@ crs:
       retreat_dist: 0.03
       approach_dist: 0.03
       tool_frame: "sander_center_link"
+      target_force: 50
     media_change:
       change_time: 300.0 # seconds
       change_pose: [-0.381, -0.202, 1.5, 0.0, -0.786, 1.57] # [px, py, pz, rx, ry, rz] in world coordinates
+      joint_names: [shoulder_pan_joint, shoulder_lift_joint, elbow_joint, wrist_1_joint, wrist_2_joint, wrist_3_joint]
+      joint_position: [-0.02467, -1.7719, -1.8680, -0.6212, 0.90265, -2.2430]
```

Comment:
How is joint_position different than change_pose?

Reply:
change pose is in Cartesian, joint_position describes the joint states. I should remove change_pose